tests/input_files/f3_bfs_xwall_8x8x8_bl1_turbinflow.dat

```diff
@@ -53,7 +53,7 @@ Switch_Step                     3
 C_Tauw                          0.011662523800000
 Min_Tauw                        0.005
 Inc_Tauw                        1.0
-Projection                      onlyl2projection
+Projection                      only_l2_projection
```

Comment:
This is a breaking change. Seems OK to me, fluid wall modeling does not have an active community, AFAIK.